tests/encryption_/models.py

```diff
@@ -0,0 +1,88 @@
+from django.db import models
+
+from django_mongodb_backend.fields import (
+    EncryptedBigIntegerField,
+    EncryptedBinaryField,
+    EncryptedBooleanField,
+    EncryptedCharField,
+    EncryptedDateField,
+    EncryptedDateTimeField,
+    EncryptedDecimalField,
+    EncryptedEmailField,
+    EncryptedFloatField,
+    EncryptedGenericIPAddressField,
+    EncryptedIntegerField,
+    EncryptedPositiveBigIntegerField,
+    EncryptedPositiveSmallIntegerField,
+    EncryptedSmallIntegerField,
+    EncryptedTextField,
+    EncryptedTimeField,
+    EncryptedURLField,
+)
+
+EQUALITY_QUERY = {"queryType": "equality"}
+RANGE_QUERY = {"queryType": "range"}
+
+
+class Appointment(models.Model):
+    time = EncryptedTimeField(queries=EQUALITY_QUERY)
+
+    class Meta:
+        required_db_features = {"supports_queryable_encryption"}
+
+
+class Billing(models.Model):
```

Comment:
Even though `EncryptedEmbeddedModel` is not supported, we can & should be able to support an EmbeddedModel object with `EncryptedField`s